static/index.js

```diff
@@ -166,6 +166,17 @@ async function displayData(_section, button) {
 
     container.innerHTML = '';
 
+    if (data.length === 0) {
+        const emptyMessage = document.createElement('div');
+        emptyMessage.textContent = 'No content available';
+        emptyMessage.style.textAlign = 'center';
```

Comment:
Move the styling part to `index.css` instead.
You can simply add a class to the message element dynamically from JS to apply the style.